doc/releases/squid.rst

```diff
@@ -0,0 +1,614 @@
+=====
+Squid
+=====
+
+Squid is the 19th stable release of Ceph.
+
+v19.2.0 Squid
+=============
+
+.. ATTENTION::
+   iSCSI users are advised that the upstream developers of Ceph encountered a
+   bug during an upgrade from Ceph 19.1.1 to Ceph 19.2.0. Read `Tracker Issue
+   68215 <https://tracker.ceph.com/issues/68215>`_ before attempting an upgrade
+   to 19.2.0.
+
+Highlights
+~~~~~~~~~~
+
+RADOS
+
+* BlueStore has been optimized for better performance in snapshot-intensive workloads.
+* BlueStore RocksDB LZ4 compression is now enabled by default to improve average performance
+  and "fast device" space usage.
+* Other improvements include more flexible EC configurations, an OpTracker to help debug mgr
+  module issues, and better scrub scheduling.
+
+Dashboard
+
+* Improved navigation layout
+
+CephFS
```

Comment:
This was specifically for CephFS improvements in the dashboard, so shouldn't it have been under the 'Dashboard' heading as it was before?

Reply:
I can move it there if you like. I'll open a new PR if you think it should go there.

Reply:
I think it should because CephFS in general will have more functionality they'd like to highlight

Reply:
I'll do it when I get back to my office.

Reply:
Thanks

Reply:
@vshankar ^

Reply:
I think I have corrected this here: https://github.com/ceph/ceph/pull/60038